packages/aws-cdk/test/integ/cli/test-cdk-failed-deploy-doesnt-hang.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+set -euo pipefail
+scriptdir=$(cd $(dirname $0) && pwd)
+source ${scriptdir}/common.bash
+# ----------------------------------------------------------
+
+setup
+
+set +e
+# this will hang if we introduce https://github.com/aws/aws-cdk/issues/6403 again.
+cdk deploy -v ${STACK_NAME_PREFIX}-failed
```

Comment:
Can we also verify that it exited with an error code, while we're at it?

Reply:
yeap